lib/experimental/stylebook/classic-screen.php

```diff
@@ -0,0 +1,44 @@
+<?php
```

Comment:
Not sure where this file should live; maybe we should put it in 6.8 compat already?

Reply:
Good idea. 

I guess we'll have to remember to come back and remove the Core action once ported?

`remove_action( 'admin_init', 'wp_add_styles_submenu_item' );`

Reply:
It'll have to stay in until we stop supporting any WP version lower than 6.7, but that's what the compat folders are for; we're supposed to audit and remove what's no longer supported after each WP release.